store.go

```diff
@@ -1031,20 +1040,21 @@ func (s *store) PutLayer(id, parent string, names []string, mountLabel string, w
 			gidMap = s.gidMap
 		}
 	}
-	var layerOptions *LayerOptions
+	layerOptions := LayerOptions{
```

Comment:
It’s not immediately obvious to me whether not passing original `options` (possibly including `TemplateLayer` down to `rlstore.Put` was intentional or not; this preserves the current behavior.